tools/run_tests/xds_k8s_test_driver/framework/xds_url_map_test_resources.py

```diff
@@ -139,6 +140,12 @@ def __init__(self, absl_flags: Mapping[str, Any] = None):
         if absl_flags is not None:
             for key in absl_flags:
                 setattr(self, key, absl_flags[key])
+        # Pick a client_namespace_suffix if not set
+        if self.resource_suffix is None:
+            self.resource_suffix = ""
+            self.client_namespace_suffix = str(int(time.time()))
```

Comment:
Just curious, why not something similar to https://github.com/grpc/grpc/blob/2dc2ef02c373ab71ddbf21e567cd83a5387df0a5/tools/run_tests/xds_k8s_test_driver/framework/xds_k8s_testcase.py#L199-L209

It produces quite readable `date-time-hash` suffix: `20210723-1101-4bp0g`

Reply:
Publicized `random_resource_suffix` and used it instead here.